go/vt/vttablet/tabletserver/query_executor_test.go

```diff
@@ -288,13 +288,15 @@ func TestQueryExecutorPlanInsertSubQuery(t *testing.T) {
 	}
 }
 
+// RBR test is almost identical to the non-RBR test, except that
+// the _stream comments are suppressed for RBR.
 func TestQueryExecutorPlanInsertSubQueryRBR(t *testing.T) {
```

Comment:
Ideally, this comment should start with `TestQueryExecutorPlanInsertSubQueryRBR` to say consistent with the rest :)

https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences doesn't really say whether it applies to test code as well or not.